lms/templates/commerce/provider.underscore

```diff
@@ -1,27 +1,26 @@
 <div class="provider-wrapper">
   <div class="provider-info">
-    <%= interpolate(
-          gettext("You still need to visit the %(display_name)s website to complete the credit process."),
-          { display_name: display_name }, true
+    <%- HtmlUtils.interpolateHtml(
+          gettext("You still need to visit the {display_name} website to complete the credit process."),
+          { display_name: display_name }
         ) %>
   </div>
   <div class="provider-more-info">
-    <%= interpolate(
-          gettext("To finalize course credit, %(display_name)s requires %(platform_name)s learners to submit a credit request."),
-          { display_name: display_name, platform_name: platformName }, true
+    <%- HtmlUtils.interpolateHtml(
+          gettext("To finalize course credit, {display_name} requires {platform_name} learners to submit a credit request."),
+          { display_name: display_name, platform_name: platformName }
         ) %>
   </div>
   <div class="provider-instructions">
-    <%= fulfillment_instructions %>
+    <%- edx.HtmlUtils.ensureHtml(fulfillment_instructions) %>
```

Comment:
Either `fulfillment_instructions` contains HTML, in which case you'd want it to be an HtmlSnippet first, and then use:
```
<%= edx.HtmlUtils.ensureHtml(fulfillment_instructions) %>
```
Or, it does not include HTML, and you would want:
```
<%- fulfillment_instructions %>
```